server/@types/viewComponents/index.d.ts

```diff
@@ -26,6 +26,7 @@ declare module 'viewComponents' {
     inductionSchedule: InductionScheduleView
     actionPlanReview: ActionPlanReviewScheduleView
     hasEditAuthority: boolean
+    userHasPermissionTo: () => boolean
```

Comment:
This is the new view model property - `userHasPermissionTo` is a function that returns a boolean.
It's use can be seen in a couple of places below